pipx/pipxrc.py

```diff
@@ -0,0 +1,155 @@
+import json
```

Comment:
rename file to something like `pipx_metadata_file.py`

Reply:
Done.